test/test_core.py

```diff
@@ -797,8 +797,8 @@ def test_mainenv(self):
     'memvalidate_verbose': ['-DEMMALLOC_MEMVALIDATE', '-DEMMALLOC_VERBOSE', '-DRANDOM_ITERS=130'],
   })
   def test_emmalloc(self, *args):
-    if '-DEMMALLOC_VERBOSE' in args and self.is_wasm64():
-      self.skipTest('EMMALLOC_VERBOSE is not compatible with wasm64')
+    self.maybe_closure()
+    self.set_setting('DEFAULT_LIBRARY_FUNCS_TO_INCLUDE', ['$ptrToString'])
```

Comment:
I don't think this line should be needed.

Can we instead add `EM_JS_DEPS(deps, "$ptrToString");` to `emmalloc.c`?

Reply:
81182932afcd553adaded8e4a7d9663cbf4861a6